lib/cli.js

```diff
@@ -557,6 +565,14 @@ internals.options = function () {
         options.sourcemaps = true;
     }
 
+    if (options.deps) {
+
+        for (let i = 0; i < options.deps.length; ++i) {
+
+            require(options.deps[i]);
```

Comment:
I think for completeness, as well as consistency with the `--transform` option and with CLIs with comparable functionality such as the nodejs CLI itself, we'll also want to support requiring files by path.  Perhaps something like this:
```suggestion
            const resolved = require.resolve(options.deps[i], { paths: [process.cwd()] });
            if (!resolved) {
                console.error(`Could not resolve module used with --require option: ${options.deps[i]}`);
                process.exit(1);
            }

            require(resolved);
```